spec/fixture_specs/failing_specs.rb

```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+require_relative "../spec_helper"
```

Comment:
I don't know why, but when running rspec via `bin/echo_exit_code.rb` rspec wouldn't pick up the normal spec_helper. 🤷

Reply:
Is the load path different when called this way?